examples/src/main/scala/zio/config/examples/autoderivation/AutoDerivationSimple.scala

```diff
@@ -42,7 +42,7 @@ object AutomaticConfig extends App {
         "aws.credentials.Token.value" -> "token",
         "port"                        -> "10",
         "default"                     -> "12",
-        "dburl"                       -> "some url",
+        "dburl.value"                 -> "some url",
```

Comment:
There is some inconsistency on the usage of value-classes. Sometimes AnyVal is extended, sometimes not. In some tests it expects the path to have the nested path, but in this code it did not. 
I think also some apps were not working in in the main branch. This change for now makes it work.